[DOC] fmriprep-docker documentation (#1515)
poldracklab/fmriprep: docs/installation.rst

<https://docs.docker.com/engine/installation/>`_.
Once Docker is installed, the recommended way to run fmriprep is to use the
fmriprep-docker_ wrapper, which requires Python and an Internet connection.
``fmriprep-docker`` is a streamlined command to run fmriprep without having to

> properly mount directories (``fmriprep-docker`` does this for you)

Reviewer: ```suggestion
properly mount directories (``fmriprep-docker`` does this for you).
```

Reviewer 2: Just to rewrite the whole sentence:

```RST
``fmriprep-docker`` is a command that allows you to write your command line
as if you were running ``fmriprep`` directly, and converts it into a ``docker``
command.
```

To install::

    $ pip install --user --upgrade fmriprep-docker

> When you run, ``fmriprep-docker`` it will generate a Docker command line for you,

Reviewer: ```suggestion
When you run ``fmriprep-docker``, it will generate a Docker command line for you,
```

> print it out for reporting purposes, and then run the command without further action

Reviewer: ```suggestion
print it out for reporting purposes, and then execute it without further action
```

needed, e.g.::

    $ fmriprep-docker /path/to/data/dir /path/to/output/dir participant
    RUNNING: docker run --rm -it -v /path/to/data/dir:/data:ro \
        -v /path/to_output/dir:/out poldracklab/fmriprep:1.0.0 \
        /data /out participant
    ...

> For ``fmriprep-docker`` all the options you would typically pass to fmriprep,

Reviewer: I'm confused by this paragraph. I think it should differentiate between docker arguments that you can pass in (e.g. environment variables and mounting dirs) and the arguments to fmriprep.

Reviewer 2: @oesteban Not sure exactly what you're suggesting here. As in we should add a section for additional `fmriprep-docker`-specific options? For that I would direct readers to [The Docker Wrapper CLI](https://fmriprep.readthedocs.io/en/stable/usage.html#the-docker-wrapper-cli), to avoid cluttering the basic "How to run fMRIPrep" page with those details.

you can use for ``fmriprep-docker``. ``fmriprep-docker`` is mostly mounting

> the paths and environmental variables.

Reviewer 2: An alternative paragraph:

```RST
``fmriprep-docker`` accepts all of the typical options for ``fmriprep``,
automatically translating directories into Docker mount points.
```

We have published a `step-by-step tutorial
<http://reproducibility.stanford.edu/fmriprep-tutorial-running-the-docker-image/>`_

> illustrating how to run ``fmriprep-docker``. This tutorial also provides valuable

Reviewer: Start sentence in a new line, down the line that is easier to maintain.

<https://docs.docker.com/engine/installation/>`_.
Once Docker is installed, the recommended way to run fmriprep is to use the
fmriprep-docker_ wrapper, which requires Python and an Internet connection.
``fmriprep-docker`` is a command that allows you to write your command line 

> as if you were running ``fmriprep`` directly, converts it into a ``docker`` 

Reviewer 2: "and converts" or "converting"

command.

To install::

    $ pip install --user --upgrade fmriprep-docker

When you run ``fmriprep-docker``, it will generate a Docker command line for you,
print it out for reporting purposes, and then execute it without further action
needed, e.g.::

    $ fmriprep-docker /path/to/data/dir /path/to/output/dir participant
    RUNNING: docker run --rm -it -v /path/to/data/dir:/data:ro \
        -v /path/to_output/dir:/out poldracklab/fmriprep:1.0.0 \
        /data /out participant
    ...

``fmriprep-docker`` accepts all of the typical options for ``fmriprep``,
automatically translating directories into Docker mount points.

We have published a `step-by-step tutorial
<http://reproducibility.stanford.edu/fmriprep-tutorial-running-the-docker-image/>`_
illustrating how to run ``fmriprep-docker``. 
This tutorial also provides valuable troubleshooting insights and advice on 
what to do after fmriprep has run.

> 

Reviewer 2: ```suggestion
```
